api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -654,6 +654,34 @@ components:
           schema:
             $ref: "#/components/schemas/Errors"
   schemas:
+    # Schemas ending in "Type" are intentionally named to prevent naming conflicts with db model classes
+    GranularityType:
+      type: string
+      enum: [ Daily, Weekly, Monthly, Quarterly, Yearly ]
+      x-enum-varnames: [ DAILY, WEEKLY, MONTHLY, QUARTERLY, YEARLY ]
+    ServiceLevelType:
+      type: string
+      enum: [ "", Premium, Standard, Self-Support, _ANY ]
+      x-enum-varnames: [ UNSPECIFIED, PREMIUM, STANDARD, SELF_SUPPORT, ANY ]
+    UsageType:
+      type: string
+      enum: [ "", Production, Development/Test, Disaster Recovery, _ANY ]
+      x-enum-varnames: [ UNSPECIFIED, PRODUCTION, DEVELOPMENT_TEST, DISASTER_RECOVERY, ANY ]
```

Comment:
I'd rather not maintain the same list twice; looking at the result without `x-enum-varnames`, we get var names like specified _except_ for `EMPTY` instead of `UNSPECIFIED` and `_ANY` instead of `ANY`. In spite of the inconsistency between `EMPTY` and `UNSPECIFIED`, I prefer to omit `x-enum-varnames` (in the other cases too). Perhaps we should just rename UNSPECIFIED to EMPTY in other places? Thoughts?

Reply:
I'm 100% in agreement.  I actually had done that first, added x-enum-varnames because of the unspecified/empty situation, then reverted because I was trying to minimize how invasive my changes were going to be.

I didn't really like using x-enum-varnames to begin with since it's a OpenAPI vendor extension.  Personally I like to keep everything pretty vanilla as long as possible.